content/en/docs/reference/access-authn-authz/node.md

```diff
@@ -71,29 +71,9 @@ and would need to continue to be authorized via whatever mechanism currently aut
 The `NodeRestriction` admission plugin would ignore requests from these kubelets,
 since the default node identifier implementation would not consider that a node identity.
 
-### Upgrades from previous versions using RBAC
-
-Upgraded pre-1.7 clusters using [RBAC](/docs/reference/access-authn-authz/rbac/) will continue functioning as-is because the `system:nodes` group binding will already exist.
```

Comment:
can we replace this content with a link to the prior version instead of just deleting it?

Reply:
How about retaining with a reword to clarify why it's still relevant:
“If your have upgraded your cluster over time, and the cluster was originally running a Kubernetes version earlier than v1.7, &hellip;”